sv-parser/src/lib.rs

```diff
@@ -136,72 +143,33 @@ pub fn parse_sv_str<T: AsRef<Path>, U: AsRef<Path>, V: BuildHasher>(
     include_paths: &[U],
     ignore_include: bool,
 ) -> Result<(SyntaxTree, Defines), Error> {
-    let (text, defines) = preprocess_str(s, path, pre_defines, include_paths, ignore_include, 0)?;
-    let span = Span::new_extra(text.text(), SpanInfo::default());
-    let result = all_consuming(sv_parser)(span);
-    match result {
-        Ok((_, x)) => Ok((
-            SyntaxTree {
-                node: x.into(),
-                text,
-            },
-            defines,
-        )),
-        Err(x) => {
-            let pos = match x {
-                nom::Err::Incomplete(_) => None,
-                nom::Err::Error(e) => error_position(&e),
-                nom::Err::Failure(e) => error_position(&e),
-            };
-            let origin = if let Some(pos) = pos {
-                if let Some(origin) = text.origin(pos) {
-                    Some((origin.0.clone(), origin.1))
-                } else {
-                    None
-                }
-            } else {
-                None
-            };
-            Err(Error::Parse(origin))
-        }
-    }
+    let (text, defines) = preprocess_str(
+        s,
+        path,
+        pre_defines,
+        include_paths,
+        ignore_include,
+        false,
+        0,
+    )?;
+    parse_sv_pp(text, defines)
 }
 
 pub fn parse_lib<T: AsRef<Path>, U: AsRef<Path>, V: BuildHasher>(
     path: T,
     pre_defines: &HashMap<String, Option<Define>, V>,
     include_paths: &[U],
+    strip_comments: bool,
```

Comment:
In my understanding, this option can be removed because user can use `preprocess` and `parse_lib_pp` to use `strip_comment`.

Reply:
Yes, I agree. Changed this in the latest commit.